### PR TITLE
Add styling for presentation demo

### PR DESCRIPTION
**Pull request by asungur**
Quick styling.
I left some in-code comments regarding the things that we might need to do.
I will also comment them here

image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -125,44 +126,52 @@ const App = () => {
...
-
-      <div>
-        Your result:
-        <p className="result">
-          {JSON.stringify(resultSection)}
-        </p>
+    <div className='h-screen'>
+      <Header />
+      <div className='flex flex-col items-center h-full' id='main'>
+        {/* PAGE /USERS */}
```
Comment (asungur) [inline src/App.js:132]: @partemen86 I've marked the pages we could have. Its just a suggestion, the split is up to you

File: src/App.js [hunk 2]
```diff
@@ -125,44 +126,52 @@ const App = () => {
...
+        <div className="flex-auto flex flex-col items-center max-w-screen-lg pt-12 pb-2 mb-64 h-full">
+          <p className="text-2xl text-black mb-2">User registration</p>
+          <UserForm onRegister={handleRegister} onLogin={handleLogin} onLogout={handleLogout}/>
+        </div>
+        {/* END OF PAGE /USERS */}
+
+        {/* PAGE /COLLECTIONS */}
+        <div className="flex-auto flex flex-col items-center max-w-screen-lg pt-12 pb-2 mb-64 h-full">
+          <p className="text-2xl text-black mb-2">Data operations</p>
+        {/* CAN WE PUT JSON DISPLAY THING AT THE TOP HERE TO SHOW RESULTANT DATA? */}
```
Comment (asungur) [inline src/App.js:142]: We need to find something to display data in for our demo.
If it is not too hard to set up, can we have the structured JSON panel you created for the `admin-frontend` @partemen86 ?
This one
image

File: src/App.js [hunk 3]
```diff
@@ -125,44 +126,52 @@ const App = () => {
...
+            <CollectionItemInput title='Create collection item' onSubmit={handleCreateItem} isJson={true}/>
+            <CollectionItemInput title='PATCH a collection item' onSubmit={handleOverwriteItem} isUpdate={true}/>
+            <CollectionItemInput title='PUT a collection item' onSubmit={handleUpdateItem} isUpdate={true}/>
+            <CollectionItemInput title='Delete collection item' onSubmit={handleDeleteItem}/>
+          </div>
+        </div>
+        {/* END OF PAGE /COLLECTIONS */}
+
+        {/* PAGE /CCFS */}
+        {/* WE CAN LOOK INTO STRIPE PAYMENT API TEST METHOD AND DEMO GETTING PAYMENT (INTERESTING CHALLENGE) */}
```
Comment (asungur) [inline src/App.js:155]: Not a prio. We just need to decide what we are going to demo for CCF.
The coolest thing we can do IMO is to get a Stripe API KEY https://stripe.com/docs/development/quickstart
Try to process a dummy payment in this app.
We can look into this later or this week if you are interested in @Reillyknut

File: src/App.js [hunk 4]
```diff
@@ -125,44 +126,52 @@ const App = () => {
...
+          <p className="text-2xl text-black mb-4">Cloud Code Functions</p>
+          <RunCcf onSubmit={handleRunCcf}/>
+        </div>
+        {/* PAGE /CCFS */}
+
+
+        {/* PAGE /FILES */}
+        <div className="flex-auto flex flex-col items-center max-w-screen-lg pt-12 pb-2 mb-64 w-1/2 h-full">
+          <p className="text-2xl text-black mb-4">Files</p>
+          {/* WE SHOULD SHOW CONTENTS OF A FILE HERE, PROBABLY AN IMAGE */}
```
Comment (asungur) [inline src/App.js:167]: I think the easiest and the most reasonable thing we can demo for the files is to upload, display an image. @Reillyknut 
Displaying JSON data is fine but file storage is for static assets usually.

File: src/components/RunCcf.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    onSubmit(ccfName, paramArray);
+    setCcfName('');
+  }
+
+  const getParams = (paramString) => {
+    return paramString.split(',').map(word => word.trim());
+  }
+
+  return (
+    // CAN WE USE FORMATTED JSON DISPLAY PANEL TO LOG THE CCF RESULT HERE?
```
Comment (asungur) [inline src/components/RunCcf.js:28]: Ignore this for now please. If we decide to do Stripe demo. We could display something else

File: tailwind.config.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  content: [
+    "./src/**/*.{js,jsx,ts,tsx}",
+  ],
+  theme: {
+    colors: {
+      transparent: 'transparent',
+      current: 'currentColor',
+      'blue1': '#445EF2',
```
Comment (asungur) [inline tailwind.config.js:9]: Same colors and font that I've used for RequestPool app

**Review by asungur — COMMENTED** [review]
I've left some notes for me/ @Reillyknut and @partemen86.
Some of these are the things Pavlo already experienced in the admin-front-end app.
One interesting thing Reilly or I can look at is to demo a Stripe payment.

**Review by partemen86 — APPROVED** [review]

**Pull request merged**